lib/morse_translator.rb

```diff
@@ -38,6 +39,27 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+  end
+
+  def translate(message)
+    holder = []
+    morse = []
+    holder << message.chars
+    @morse_array = holder.flatten
+    @morse_array.each do |char, value|
+      morse << @dictionary[char] unless char == char.upcase
+        morse << capitalized_dictionary[char]
```

Comment:
Another way to do this would be to downcase the character before you put it in the @dictionary hash:

```ruby
morse << @dictionary[char.downcase]
```